weave/trace_server/opentelemetry/constants.py

```diff
@@ -53,6 +53,8 @@
     "mlflow.spanInputs",  # From MLFlow's tracking format
     "traceloop.entity.input"  # From Traceloop's conventions
     "input",  # Generic fallback for Pydantic models - lowest priority
+    "gcp.vertex.agent.llm_request",
+    "gcp.vertex.agent.tool_call_args",
```

Comment:
Let's put these above `input`. It's probably fine in this scenario but `input` should be lowest priority.

Just to confirm:
Once `gcp.vertex.agent.llm_request` is found `gcp.vertex.agent.tool_call_args` won't be checked for.

If both are present in the call this will require some changes so that we can parse out both. Is there a scenario when we would need to parse out both of these values for a single Span?

Reply:
I've added these before the input, also changed the ordering. In my tests I didn't come across a scenario where both need to be parsed